web2/htdocs/index.html

```diff
@@ -243,6 +243,36 @@ <h2>CLI Authentication Credentials</h2>
             </div>
           </div>
     <!-- }}} --></script>
+    <script type="text/html" id="tpl--disasterkey"><!-- {{{ -->
+        [[#
+          {disasterkey} template
+  
+          Displays the Shield Disaster Key, so that operators can restore
+          fixed-key backups in the event of a disaster.
+        ]]
+            <div id="cliauth" class="frontdoor">
```

Comment:
This shouldn't be "cliauth" - probably need to rework CSS if it doesn't fall under `.frontdoor`